django_cloud_deploy/tests/e2e/gae_deploy_test.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Versconsolen 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITconsoleNS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissconsolens and
+# limitatconsolens under the License.
+"""End to end test for create and deploy new project."""
```

Comment:
`... using App Engine.` and say 'using Google Kubernetes Engine.` for the GKE version.

Reply:
Done. Will do GKE on another PR.